docs/ref/middleware.txt

```diff
@@ -158,12 +158,22 @@ It will NOT compress content if any of the following are true:
 You can apply GZip compression to individual views using the
 :func:`~django.views.decorators.gzip.gzip_page()` decorator.
 
+.. note::
+
+    If the response to be compressed has an ``ETag`` header, the header will
+    be changed to a weak ``ETag`` to comply with :rfc:`7232`.
+
 .. versionchanged:: 1.10
 
     In older versions, Django's CSRF protection mechanism was vulnerable to
     BREACH attacks when compression was used. This is no longer the case, but
     you should still take care not to compromise your own secrets this way.
 
+.. versionchanged:: 1.11
+
+    In older versions, the ``ETag`` header was changed in a way that did not
```

Comment:
My intention was for the "If the response..." part to be self-sufficient since the `versionchanged` stuff will eventually disappear. So if you think it might be confusing we should change it there.

I doubt that anyone will notice or care about the specific changes to the ETag modification. The main reason I see to document the change itself (whether here with `versionchanged` or in the release notes) is because this could actually cause a large performance improvement (reduction in used bandwidth) for some sites (that use large gzipped responses and conditional requests), and such people might be interested in tracking down the reason. If that seems compelling to you, let's leave it in (or put it in the release notes), if not we can probably leave it out.